runtime/src/main/java/org/corfudb/protocols/wireprotocol/WriteRequest.java

```diff
@@ -12,25 +12,15 @@
  */
 @Builder
 @AllArgsConstructor
-public class WriteRequest implements ICorfuPayload<WriteRequest>, IMetadata {
+public class WriteRequest implements IMetadata {
 
     @Getter
```

Comment:
You can just eliminate this class and Just use the LogData object directly.

Reply:
Done.